### ISSUE TITLE
What is the training setting for reproducing ShuffleNetV2+？

### ISSUE THREAD

**Issue by jianghaojun** [megvii-model/ShuffleNet-Series#42]
I am curious about the training setting for reproducing ShuffleNetV2+：

1. Iteration numbers？According to the train.py code, total iteration is 450000.
1. Data Augmentation？Did you use less aggressive scale data augmentation？According to the paper of ShuffleNetV1, authors mentioned - we use slightly less aggressive scale augmentation for data preprocessing. Similar modifications are also referenced in MobileNet because such small networks usually suffer from underfitting rather than overfitting. If you use the less scale data augmentation, then what is the scale range (default = (0.08, 1.00))?

**Comment by nmaac** [comment]
1. yes 450k
2. no shufflenetv2+ does not suffer from the under-fitting problem
